framework/packages/abstract-std/src/native/ibc_host.rs

```diff
@@ -136,6 +136,7 @@ pub enum ExecuteMsg {
         source_module: InstalledModuleIdentification,
         target_module: ModuleInfo,
         msg: Binary,
+        is_query: bool,
```

Comment:
Either rename enum type to `ModuleAction` or create a new variant `ModuleQuery`. 

If ModuleAction then `msg` can be an enum 

```rust
ModuleMsg {
   Execute(Binary)
   Query(Binary)
}
```

Reply:
Raw could be useful, not sure about the rest

Reply:
Also name could be confusing, because we already have `ModuleIbcMsg` should it be `ModuleAction` instead?

Reply:
> Raw could be useful, not sure about the rest

Yeah let's stick to Binary msg (smart query). We can add raw as a module action later if there's a request for it (but it's better solved with ICQ anyway)

Reply:
And `ModuleAction` sounds good!!